cmake/thirdparty/SetupMPI.cmake

```diff
@@ -177,10 +177,24 @@ if (ENABLE_FORTRAN)
     endif()
 endif()
 
-blt_import_library(NAME          mpi
-                   INCLUDES      ${_mpi_includes}
-                   TREAT_INCLUDES_AS_SYSTEM ON
-                   LIBRARIES     ${_mpi_libraries}
-                   COMPILE_FLAGS ${_mpi_compile_flags}
-                   LINK_FLAGS    ${_mpi_link_flags} 
-                   EXPORTABLE    ${BLT_EXPORT_THIRDPARTY})
+
+if(BLT_USE_MPI_IMPORTED_TARGETS)
```

Comment:
imported is a bit overloaded in this context.

Reply:
maybe "BLT_USE_CMAKE_MPI_TARGETS"?

Reply:
the other MPI options are all have the `BLT_MPI` prefix, like `BLT_MPI_COMPILE_FLAGS` 

how about:
`BLT_MPI_USE_FINDMPI_CMAKE_TARGETS`

Reply:
That works.. after this is proven to solve you problem, we should handle the overriding case where the user provides includes/link flags/etc.

Reply:
or 
`BLT_MPI_USE_FIND_MPI_CMAKE_TARGETS`

Reply:
BLT_USE_FIND_MPI_TARGETS?